### PR TITLE
[zh] Tweak admission controller page

### PR DESCRIPTION
**Pull request by tengqm**
This PR updates the line wrapping of the admission-controllers page and fixes some nits found in the existing translation.

### CODE REVIEW

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -394,18 +394,20 @@ Services which use `externalIPs` and may not add new values to `externalIPs` on
 existing `Service` objects.  Existing uses of `externalIPs` are not affected,
 and users may remove values from `externalIPs` on existing `Service` objects.
 -->
-该准入控制器拒绝 `Service` 字段 `externalIPs` 的所有新规使用。 此功能非常强大（允许网络流量拦截），
-并且无法很好地受策略控制。 启用后，群集用户将无法创建使用 `externalIPs` 的新服务，也无法在现有
-`Service` 对象上向 `externalIPs` 添加新值。 `externalIPs` 的现有使用不受影响，用户可以从现有
-`Service` 对象上的 `externalIPs` 中删除值。
+此准入控制器拒绝新的 `Service` 中使用字段 `externalIPs`。
+此功能非常强大（允许网络流量拦截），并且无法很好地受策略控制。
+启用后，群集用户将无法创建使用 `externalIPs` 的新 `Service`，也无法在现有
```
Comment (Sea-n) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:399]:
```suggestion
启用后，集群用户将无法创建使用 `externalIPs` 的新 `Service`，也无法在现有
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -344,8 +344,8 @@ This admission controller will deny exec and attach commands to pods that run wi
 allow host access.  This includes pods that run as privileged, have access to the host IPC namespace, and
 have access to the host PID namespace.
 -->
-该准入控制器将拒绝在由于拥有升级特权，而具备访问宿主机能力的 Pod 中执行 exec 和
-attach 命令。这包括在特权模式运行的 Pod，可以访问主机 IPC 名字空间的 Pod，
+此准入控制器将拒绝在由于拥有提级特权而具备访问宿主机能力的 Pod 中执行 exec 和
+attach 命令。这类 Pod 包括在特权模式运行的 Pod、可以访问主机 IPC 名字空间的 Pod、
 和访问主机 PID 名字空间的 Pod 。
```
Comment (Sea-n) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:349]:
```suggestion
和访问主机 PID 名字空间的 Pod。
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 3]
```diff
@@ -1091,11 +1085,12 @@ metadata:
 
 <!--
 #### Internal Behavior
+
 This admission controller has the following behavior:
 -->
 #### 内部行为
```
Comment (Sea-n) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1091]:
```suggestion
#### 内部行为  {#nternal-behavior}
```

**Review by Sea-n — COMMENTED** [review]
Here is some nitpick.

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 4]
```diff
@@ -415,30 +417,30 @@ of it.
...
 <!--
 * Enabling the `EventRateLimit` admission controller;
 * Referencing an `EventRateLimit` configuration file from the file provided to the API
   server's command line flag `--admission-control-config-file`:
 -->
 * 启用 `EventRateLimit` 准入控制器；
-* 从文件中引用 `EventRateLimit` 配置文件，并提供给 API 服务器命令的
-  `--admission-control-config-file` 标志：
+* 在通过 API 服务器的命令行标志 `--admission-control-config-file` 设置的文件中，
+  引用 `EventRateLimit` 配置文件，：
```
Comment (TinySong) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:429]:
```suggestion
  引用 `EventRateLimit` 配置文件：
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 5]
```diff
@@ -415,30 +417,30 @@ of it.
...
-- name: EventRateLimit
-  path: eventconfig.yaml
-...
-```
+  ```yaml
+  apiVersion: apiserver.config.k8s.io/v1
+  kind: AdmissionConfiguration
+  plugins:
+    - name: EventRateLimit
+      path: eventconfig.yaml
```
Comment (TinySong) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:436]:
```suggestion
  plugins:
  - name: EventRateLimit
    path: eventconfig.yaml
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 6]
```diff
@@ -601,13 +601,13 @@ clusters:
 - name: name-of-remote-imagepolicy-service
   cluster:
     certificate-authority: /path/to/ca.pem    # CA 用于验证远程服务
-    server: https://images.example.com/policy # 要查询的远程服务的 URL。必须是 'https' 。
+    server: https://images.example.com/policy # 要查询的远程服务的 URL，必须是 'https' 。
```
Comment (TinySong) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:604]:
```suggestion
    server: https://images.example.com/policy # 要查询的远程服务的 URL，必须是 'https'。
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md (hunk 5 above)
Comment (tengqm) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md:436]: These lines are indented for ease of reading.

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 7]
```diff
@@ -82,14 +82,14 @@ corresponding reclamation or reconciliation process, as a given admission
 controller does not know for sure that a given request will pass all of the
 other admission controllers.
 -->
-最后，除了对对象进行变更外，准入控制器还可以有其它作用：将相关资源作为请求处理的一部分进行变更。
-增加使用配额就是一个典型的示例，说明了这样做的必要性。
+最后，除了对对象进行变更外，准入控制器还可以有其它副作用：将相关资源作为请求处理的一部分进行变更。
```
Comment (TinySong) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:85]:
```suggestion
最后，除了会改变所涉及的对象之外，准入控制器有时也会产生副作用，即将相关资源作为请求处理的一部分进行变更。
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 8]
```diff
@@ -928,16 +922,17 @@ and enforces kubelet modification of labels under the `kubernetes.io/` or `k8s.i
   * `failure-domain.beta.kubernetes.io/zone`  (已弃用）
```
[the remark this answers is not in the document]
Comment (Sea-n) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md:922]: Hey @tengqm, I think maybe you missed this comment.
It's two space + inconsistent `(quote）`.

[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md:922]: Doest this space matter that much?

[the remark this answers is not in the document]
Comment (Sea-n) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md]: I don't care about the space, but I think the inconsistent half-width and full-width quote should not be here.

**Comment by Sea-n** [comment]
Thanks for all the hard work and fixing our nitpicks.
I think @TinySong and I are both satisfied with all changes!
/lgtm

BTW, @TinySong maybe you can take a look at [applying for the membership](https://github.com/kubernetes/community/blob/master/community-membership.md#member). 🙃

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 9]
```diff
@@ -98,23 +98,23 @@ configured with the right set of admission controllers is an incomplete server a
...
+## 如何启用一个准入控制器？  {how-do-i-turn-on-an-admission-controller}
 
 <!--
 The Kubernetes API server flag `enable-admission-plugins` takes a comma-delimited list of admission control plugins to invoke prior to modifying objects in the cluster.
 For example, the following command line enables the `NamespaceLifecycle` and the `LimitRanger`
 admission control plugins:
 -->
-Kubernetes API 服务器的 `enable-admission-plugins` 标志接受一个用于在集群修改对象之前
-调用的（以逗号分隔的）准入控制插件顺序列表。
+Kubernetes API 服务器的 `enable-admission-plugins` 标志接受一个（以逗号分隔的）准入控制插件列表,
```
Comment (howieyuen) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:114]:
```suggestion
Kubernetes API 服务器的 `enable-admission-plugins` 标志接受一个（以逗号分隔的）准入控制插件列表，
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 10]
```diff
@@ -893,51 +887,52 @@ This admission controller limits the `Node` and `Pod` objects a kubelet can modi
...
+  这类前缀的标签时保留给管理员的，用以为 `Node` 对象设置标签以隔离工作负载，而不允许 kubelet
   修改带有该前缀的标签。
-* **允许** kubelet 添加/删除/更新这些和这些前缀的标签：
+* **允许** kubelet 添加、删除、更新以下标签：
   * `kubernetes.io/hostname`
   * `kubernetes.io/arch`
   * `kubernetes.io/os`
   * `beta.kubernetes.io/instance-type`
   * `node.kubernetes.io/instance-type`
   * `failure-domain.beta.kubernetes.io/region` （已弃用）
```
Comment (howieyuen) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:921]:
```suggestion
  * `failure-domain.beta.kubernetes.io/region`（已弃用）
```

**Review by howieyuen — CHANGES_REQUESTED** [review]
found 2 nits 👀

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md (hunk 10 above)
Comment (tengqm) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md:921]: 这里这个空格有什么关系么？

**Comment by TinySong** [comment]
> Thanks for all the hard work and fixing our nitpicks. I think @TinySong and I are both satisfied with all changes! /lgtm
> 
> BTW, @TinySong maybe you can take a look at [applying for the membership](https://github.com/kubernetes/community/blob/master/community-membership.md#member). 🙃

yeah, I am willing to be a Kubernetes member,  @tengqm @Sea-n are you willing to be my sponsors? 😄

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md (hunk 10 above)
Comment (howieyuen) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md:921]: 中文括号渲染结果比较宽，就好像是有已经有了一个空格……
额，当我没说吧，毕竟不符合规范了

**Pull request merged**